pkg/cli/image/mirror/mirror.go

```diff
@@ -149,6 +150,7 @@ func NewCmdMirrorImage(name string, streams genericclioptions.IOStreams) *cobra.
 	flag.BoolVar(&o.SkipMount, "skip-mount", o.SkipMount, "Always push layers instead of cross-mounting them")
 	flag.BoolVar(&o.SkipMultipleScopes, "skip-multiple-scopes", o.SkipMultipleScopes, "Some registries do not support multiple scopes passed to the registry login.")
 	flag.BoolVar(&o.Force, "force", o.Force, "Attempt to write all layers and manifests even if they exist in the remote repository.")
+	flag.BoolVar(&o.ForceManifestList, "force-manifest-list", o.ForceManifestList, "Always attempt to mirror manifestlist even if manifestlist contains a single image digest")
```

Comment:
I am a bit concerned here that the default behavior of image mirror is wrong when users specify `--filter-by-os=*`, regardless of this flag. Mirror doesn't have any issue dealing with manifest lists, so I think ProcessManifestList within mirror should always have the equivalent of `force=true`.  That being said, mirror for release payloads *should* resolve image streams, so it can't be the default.

That may be a behavior change, but at least on the surface this looks like a bug that mirror by default strips manifest lists.

Reply:
@smarterclayton so you're basically saying the fix is correct, but we should not expose that outside for users, @kevinrizza would you mind updating?